composer.json

```diff
@@ -16,8 +16,8 @@
     "require": {
         "php": "^8.0.2",
         "ext-dom": "*",
-        "illuminate/mail": "^9.0",
-        "illuminate/support": "^9.0",
+        "illuminate/mail": "^9.0||^10.0",
+        "illuminate/support": "^9.0||^10.0.0",
```

Comment:
Could you make this inline with the other constraints? Like ^10.0

Reply:
```suggestion
      "illuminate/support": "^9.0||^10.0",
```